<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>batch</class>
 <widget class="QWidget" name="batch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Batch processing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="layout_names"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_status"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pb_progress">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>25</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
